cmd/odf/set/log_level.go

```diff
@@ -0,0 +1,50 @@
+package set
+
+import (
+	"fmt"
+	"regexp"
+
+	"github.com/red-hat-storage/odf-cli/cmd/odf/root"
+	"github.com/red-hat-storage/odf-cli/pkg/rook/common"
+	"github.com/spf13/cobra"
+)
+
+var logLevelCmd = &cobra.Command{
```

Comment:
```suggestion
var setLogLevelCmd = &cobra.Command{
```
 This will make it easier to understand what is the subcommand also this naming is followed in `kubectl` IIRC